config/packages/security.yaml

```diff
@@ -1,5 +1,4 @@
 security:
-    always_authenticate_before_granting: true
```

Comment:
this flag is deprecated in Symfony 5.4 and will be removed in Symfony 6.0. what is more, it breaks application after bumping the version to Symfony 5.4
references:
- https://symfony.com/doc/5.4/reference/configuration/security.html#always-authenticate-before-granting
- https://github.com/symfony/symfony/issues/44337

Reply:
Can we test this behaviour? https://github.com/Sylius/Sylius/pull/10289#issuecomment-480249379

Still, this is something that should be placed in UPGRADE file for others to make an upgrade, as this change will not take any effect over 3rd party users